x/tokenregistry/types/genesis.go

```diff
@@ -30,7 +37,38 @@ func GetGenesisStateFromAppState(marshaler codec.JSONCodec, appState map[string]
 	return genesisState
 }
 
-func InitialRegistry() Registry {
+func InitialAdminAccounts() *AdminAccounts {
+	return &AdminAccounts{
+		AdminAccounts: []*AdminAccount{
+			{
+				AdminType:    AdminType_CLPDEX,
+				AdminAddress: "sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd",
+			},
+			{
+				AdminType:    AdminType_CLPDEX,
+				AdminAddress: "sif1l7hypmqk2yc334vc6vmdwzp5sdefygj2ad93p5",
+			},
+			{
+				AdminType:    AdminType_PMTPREWARDS,
+				AdminAddress: "sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd",
+			},
+			{
+				AdminType:    AdminType_PMTPREWARDS,
+				AdminAddress: "sif1l7hypmqk2yc334vc6vmdwzp5sdefygj2ad93p5",
+			},
+			{
+				AdminType:    AdminType_ETHBRIDGE,
+				AdminAddress: "sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd",
+			},
+			{
+				AdminType:    AdminType_TOKENREGISTRY,
+				AdminAddress: "sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd",
+			},
```

Comment:
@Jedi2002 is this sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd the real admin account? or do we still need to update the migrator?

Reply:
WE need to add to the migrator as well , I was thinking of an if condition like so 
if ctx.ChainID() != "sifchain-1" { } 
if ctx.ChainID() != "sifchain-devnet-1" { } 
if ctx.ChainID() != "sifchain-testnet-1" { } 

Just need the correct addresses for that